### ISSUE TITLE
What is the best parameter set to train using ade20k dataset?

### ISSUE THREAD

**Issue by tan-z-tan** [hellochick/PSPNet-tensorflow#18]
Thanks for useful repository!

I'm trying to train PSPNet from scratch using ade20k dataset.
It works by changing train.py like this:
```
DATA_DIRECTORY = '../data/ADEChallengeData2016/'
DATA_LIST_PATH = './list/ade20k_train_list.txt'
#IGNORE_LABEL = 255
IGNORE_LABEL = 0

#INPUT_SIZE = '713,713'
INPUT_SIZE = '473,473'

#NUM_CLASSES = 19
NUM_CLASSES = 150
```

However, the model i got after 10000 steps has poor performance. (average training loss reached around 2.5 from 5 to 6)
I guess the parameter set written in train.py is set to train cityscapes dataset.
What is the best parameter set to train using ade20k dataset?

**Comment by TingZhang08** [comment]
I also encounter this problem. Have you resolved?

**Comment by shipengai** [comment]
Hi，@tan-z-tan
   can you share your training on ade20k?
 thanks
